apps/src/music/player/SamplePlayer.ts

```diff
@@ -1,5 +1,5 @@
+import { Effects } from './interfaces/Effects';
```

Comment:
Have our linter rules diverged slightly, with spaces now appearing?

Reply:
Oh good catch, looks like it's not checking TS files! Let me see if there's a quick fix for that.

Reply:
Will be slightly more involved, so I'll do a fast follow for this